src/examples/simple_snapp.mjs

```diff
@@ -0,0 +1,85 @@
+// this is a pure JS version of the simple_snapp.ts example, that could be the starting point
```

Comment:
Great idea to include this!

Reply:
nit: should this be a doccomment? `///` , not sure exactly how that gets surfaced by tools when it's at the top of a file like this, but seems "doccommentish" to me

Reply:
I removed this comment at some point. To have some docs for pure JS usage, I instead chose to doc-comment the two functions that enable using decorator functionality without decorators: https://github.com/o1-labs/snarkyjs/pull/123/commits/fb01a0970d1fc102d9eea5ab8614b7fd7410255e